src/main/java/seedu/address/model/person/UniquePersonList.java

```diff
@@ -97,6 +100,27 @@ public void setPersons(List<Person> persons) {
         internalList.setAll(persons);
     }
 
+    /**
+     * Adds a tag to a person in the list.
+     * The person must exist in the list.
+     */
+    public void removeTagFromPerson(Person target, Set<Tag> tags) {
```

Comment:
Just to clarify since this method is removing a tag from a person, is it possible for it to be implemented in some other class? Since a Person should have the responsibility of removing a tag, and this method should not need to be exposed to the UniquePersonList.

Reply:
Ok i realized i dont need this function at all so i removed it